src/docs/truffle/quickstart.md

```diff
@@ -285,7 +285,7 @@ The only extra step, aside from running Ganache, is that it requires editing the
 
 1. Download and install [Ganache](/ganache).
 
-1. Open `truffle.js` in a text editor. Replace the content with the following:
+1. Open `truffle-config.js` or `truffle.js` in a text editor. Replace the content with the following:
```

Comment:
Can you just delete the "or `truffle.js`" part of this sentence? Going forward we are only going to use `truffle-config.js`.

Reply:
@eggplantzzz done.

Reply:
Awesome, thanks!